app/views/proposals/detail/_request_details.html.haml

```diff
@@ -0,0 +1,85 @@
+                    = @client_data_instance.description
+
+
+            - proposal.fields_for_input.each do |fields|
+              - if !excluded_values.include? fields[:key]
+                .row
+                  .medium-12.column
+                    - field = fields
+                    - if field
+                      .detail-wrapper
+                        .detail-display
+                          %label.detail-element
+                            = field[:value][0]
+                          %span.detail-value{id: field[:key] + '-' + @client_data_instance.id.to_s }
+                            = request_details_form_value field[:value][1]
+                        .detail-edit
+                          %label.detail-element
+                            = field[:value][0]
+                            = request_details_form_field field[:key], f
+
+            .row.column.line-item-block
+              .medium-12.callout.primary.column.medium-centered
+                .row
+                  .medium-12.column
+                    .detail-wrapper
+                      %label.detail-element Vendor
+                      .detail-edit
+                        = f.input :vendor,
+                          input_html: { class: "js-selectize required", data: { initial: JSON.generate(vendor_options(@client_data_instance.vendor)) }}
+                      .detail-display
+                        %span.detail-value{id: 'vendor-' + @client_data_instance.id.to_s }
+                          = @client_data_instance.vendor || "-"
+                    .detail-wrapper
+                      %label.detail-element CL number
+                      .detail-edit
+                        = f.input :cl_number, label: false
+                      .detail-display
+                        %span.detail-value{id: 'cl_number-' + @client_data_instance.id.to_s }
+                          = @client_data_instance.cl_number || "-"
+
+
+                    .detail-wrapper
+                      %label.detail-element Building number
+                      .detail-edit
+                        = f.input :building_number,
+                          input_html: { class: "js-selectize required", data: { initial: JSON.generate(building_options) }}
+                      .detail-display
+                        %span.detail-value{id: 'building_number-' + @client_data_instance.id.to_s }
+                          = @client_data_instance.building_number
+            .row.column.line-item-block
+              .medium-12.primary.column.medium-centered
+                .row
+                  .medium-6.column
+                  .medium-6.column
+                    .primary.callout.detail-wrapper.no-margin-bottom.no-padding-bottom
+                      .detail-edit
+                        = f.input :amount, as: :currency, input_html: { value: sprintf("%.2f", @client_data_instance.amount) }
```

Comment:
we have this method in `app/helpers/value_helper.rb` (`property_to_s`) that returns decimal values as currency -- is that what you want here? This method is also used in `property_to_s` referenced above, so if we used that on all client data attrs this would show up as currency